django_geosource/models.py

```diff
@@ -162,7 +163,9 @@ def update_fields(self):
 
                 if field_name not in fields:
                     field, is_new = self.fields.get_or_create(
-                        name=field_name, defaults={"label": field_name}
+                        name=field_name,
+                        defaults={"label": field_name},
+                        order=len(fields.items()) + 1,
```

Comment:
Use enumerate instead

Reply:
Yes, you're right. I didn't know you could enumerate on a dict (even if it makes absolute sense in python the more I think about it :D).

Reply:
I'm not sure to understand, we are not "looping" through `fields`

Reply:
Yeah but you loop through record.items() i think ?

Reply:
Done.